Common/FeatureFlags.swift

```diff
@@ -18,6 +18,7 @@ struct FeatureFlagConfiguration: Decodable {
     let fiaspInsulinModelEnabled: Bool
     let includeServicesInSettingsEnabled: Bool
     let manualDoseEntryEnabled: Bool
+    let insulinDeliveryReservoirEnabled: Bool
```

Comment:
[nit] This is a bit of a misnomer. It's really enabling/disabling the insulin delivery reservoir *view* (or tab, I suppose) not the insulin delivery reservoir feature itself. That is still being used internally.

Reply:
Good point!  Will change